apps/ledger-live-desktop/src/renderer/components/DeviceAction/NoSuchAppOnProviderErrorComponent.tsx

```diff
@@ -95,12 +97,14 @@ const NoSuchAppOnProviderErrorComponent: React.FC<{
         >
           <Trans i18nKey="errors.NoSuchAppOnProvider.exploreCTA" />
         </StyledLinkExploreButton>
-        <StyledLinkLearnMoreButton
-          onClick={handleOnOpenExternalLink(urls.learnMoreLedgerSync)}
-          Icon={renderExploreIcon}
-        >
-          <Trans i18nKey="errors.NoSuchAppOnProvider.learnMoreCTA" />
-        </StyledLinkLearnMoreButton>
+        {learnMoreLink && (
```

Comment:
as `learnMoreTextKey` is optional, needs to be checked no?

Reply:
why not ! done

Reply:
thanks

Reply:
@mcayuelas-ledger can you take a look please ?